src/Text.js

```diff
@@ -89,7 +88,7 @@ function extend (Y) {
             } else if (event.type === 'delete') {
               let start = aceDocument.indexToPosition(event.index, 0)
               let end = aceDocument.indexToPosition(event.index + event.length, 0)
-              var range = new Range(start.row, start.column, end.row, end.column)
+              var range = { start, end }
```

Comment:
I'm unsure if it is a good idea to "imitate" the Range object. I can't find any documentation for this.